extensions/wikia/Listusers/SpecialListusers_ajax.php

```diff
@@ -41,65 +45,140 @@ public static function axShowUsers ( ) {
+					$username .= "<br />";
+					$username .= Xml::tags( 'span', array( 'style' => 'font-size:77%; padding-left:5px;' ), $data['links'] );
+					$username .= Xml::closeElement('div');
 
-			if ( !empty( $records ) && is_array( $records ) ) {
-				$result['iTotalRecords'] = intval( $limit );
-				$result['iTotalDisplayRecords'] = intval($records['cnt']);
-				$result['sColumns'] = $records['sColumns'];
-				$rows = array();
-				if ( isset($records['data'] ) ) {
-					foreach ( $records['data'] as $user_id => $data ) {
-						$username  = Xml::openElement('div', array( 'class' => ( $data['blcked'] ) ? 'listusers_blockeduser' : '' ) );
-						$username .= Xml::tags( 'span', array( 'style' => 'font-size:90%;font-weight:bold;padding-left:5px;' ), $data['user_link'] );
-						$username .= "<br />";
-						$username .= Xml::tags( 'span', array( 'style' => 'font-size:77%; padding-left:5px;' ), $data['links'] );
-						$username .= Xml::closeElement('div');
-
-						$groups = ( $data['blcked'] ) ? Xml::tags( 'span', array( 'class' => 'listusers_blockeduser' ), $data['groups'] ) : $data['groups'];
-						$edits = ( $data['blcked'] ) ? Xml::tags( 'span', array( 'class' => 'listusers_blockeduser' ), $data['rev_cnt'] ) : $data['rev_cnt'];
-
-						$last_edited  = "-";
-						if ( $data['last_edit_ts'] && $data['last_edit_page'] ) {
-							$last_edited  = Xml::openElement( 'div' );
-							$last_edited .= Xml::tags( 'span',
-								array( 'style' => 'font-size:90%;' ),
-								Xml::element('a', array( 'href' => $data['last_edit_page'] ), $data['last_edit_ts'] )
-							);
-							$last_edited .= Xml::tags( 'span',
-								array( 'style' => 'font-size:77%; padding-left:8px;' ),
-								Xml::element('a', array( 'href' => $data['last_edit_diff'] ), wfMsg('diff') )
-							);
-							$last_edited .= Xml::closeElement('div');
-						}
-
-						$rows[] = array(
-							$username, //User name
-							$groups, //Groups
-							$edits,//Revisions (edits)
-							$last_edited//Last edited
+					$groups = ( $data['blcked'] ) ? Xml::tags( 'span', array( 'class' => 'listusers_blockeduser' ), $data['groups'] ) : $data['groups'];
+					$edits = ( $data['blcked'] ) ? Xml::tags( 'span', array( 'class' => 'listusers_blockeduser' ), $data['rev_cnt'] ) : $data['rev_cnt'];
+
+					$last_edited  = "-";
+					if ( $data['last_edit_ts'] && $data['last_edit_page'] ) {
+						$last_edited  = Xml::openElement( 'div' );
+						$last_edited .= Xml::tags( 'span',
+							array( 'style' => 'font-size:90%;' ),
+							Xml::element('a', array( 'href' => $data['last_edit_page'] ), $data['last_edit_ts'] )
+						);
+						$last_edited .= Xml::tags( 'span',
+							array( 'style' => 'font-size:77%; padding-left:8px;' ),
+							Xml::element('a', array( 'href' => $data['last_edit_diff'] ), wfMsg('diff') )
 						);
+						$last_edited .= Xml::closeElement('div');
 					}
+
+					$rows[] = array(
+						$username, //User name
+						$groups, //Groups
+						$edits,//Revisions (edits)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app